package.json

```diff
@@ -39,7 +39,7 @@
     "@types/node": "^13.7.1",
     "apollo-server": "^2.10.0",
     "dedent": "^0.7.0",
-    "graphql": "^14.6.0",
+    "graphql": "^15.4.0",
```

Comment:
this is a devDependency I don't think it will cause version conflicts when installing this library

maybe you could update the peerDependency instead and make it v14 or v15

Reply:
fixed it myself